lib/shopify_app/test_helpers/webhook_verification_helper.rb

```diff
@@ -8,7 +9,7 @@ def authorized_webhook_verification_headers!(params = {})
         @request.headers['HTTP_X_SHOPIFY_HMAC_SHA256'] = valid_hmac
       end
 
-      def unauthorized_webhook_verification_headers!(params = {})
+      def unauthorized_webhook_verification_headers!
```

Comment:
I can't find where is this method called from?

Maybe it sometimes passes `params` hash and sometimes it doesn't. In case it "sometimes does", this method will now raise error.

Can you please double check that this works as expected?

/cc @tanema how/where is this method called from?

Reply:
These methods were introduced in https://github.com/Shopify/shopify_app/pull/950 and are helper methods meant to be called from the main application tests.